packages/rpc/src/binance/BinanceRPC.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { getEnv } from '../utils';
+import { BinanceRPC } from './BinanceRPC';
+import { BinanceNodeInfo } from './models';
+
+describe('BinanceRPC', () => {
+    
+    let rpc: BinanceRPC;
+    let address: string;
+    let binanceRpcUrl: string
+    let marketLimit: number
+    let marketOffset: number
+    let order: string
+    let symbol: string
+    let transactionHash: string
+    let sync: string
+
+    beforeEach(function() {
+        require('dotenv').config({ path: __dirname + '/../.env' });
+        binanceRpcUrl = getEnv('BinanceRPC');
```

Comment:
One more thing regarding naming convention. Please use uppercased env names. Like you may find in other packages. `BinanceRPC ` should be `BINANCE_RPC_URL`.